site/src/comparison.rs

```diff
@@ -257,18 +259,30 @@ pub async fn compare_given_commits(
     }))
 }
 
-/// Data associated with a specific date
+/// Data associated with a specific artifact
 #[derive(Debug, Clone, Serialize)]
 pub struct DateData {
```

Comment:
This should probably be renamed `ArtifactData` as this is for a given artifact which may or may not have a date associated with it.

Reply:
Seems reasonable. It was named prior to the notion of artifacts -- and "CommitData" was already used for something else I think at that time...